projects/mupdf.com/package.yml

```
@@ -0,0 +1,42 @@
+distributable:
+  url: git+https://github.com/ArtifexSoftware/mupdf
+  ref: ${{version}}
+  strip-components: 1
+
+versions:
+  github: ArtifexSoftware/mupdf/tags
+
+dependencies:
+  zlib.net: '*'
+  linux:
+    freeglut.sourceforge.io: '*'
+    freedesktop.org/mesa-glu: '*'
+    x.org/protocol: '*'
+    x.org/x11: '*'
+    x.org/xcursor: '*'
+    x.org/xinerama: '*'
+    x.org/xrandr: '*'
+    mesa3d.org: '*'
+    gnome.org/glib: '*'
+    x.org/xtrans: '*'
+
+build:
+  dependencies:
+    git-scm.org: '*'
+    gnu.org/binutils: '*'
```

You'll probably also need `gnu.org/gcc` for `g++`

```suggestion
    gnu.org/binutils: '*'
    gnu.org/gcc: '*'
```